### ISSUE TITLE
[Bug] Frame IsClippedToBound broken on IOS somewhere after 4.6.0.800

### ISSUE THREAD

**Issue by ish-1313** [xamarin/Xamarin.Forms#11278]
### Description
I have the following code (related part)
 ```
     <Grid Padding="0" ColumnSpacing="2" Margin="0,5,0,5" HorizontalOptions="Fill" x:Name="Item">
                            <Grid.ColumnDefinitions>
                                <ColumnDefinition Width="{StaticResource PetImageWidth}"/>
                                <ColumnDefinition Width="*" />
                            </Grid.ColumnDefinitions>
                            <Grid.RowDefinitions>
                                <RowDefinition Height="{StaticResource PetImageHeight}" />
                            </Grid.RowDefinitions>

                            <Frame Grid.Row="0" Grid.Column="0"  IsClippedToBounds="True" CornerRadius="50"   Padding="0" 
                                                   HorizontalOptions="Fill" VerticalOptions="Fill">
                                <Image   Source="{Binding BindingImageSource}" Style="{StaticResource PetImageStyle}"  />
                            </Frame>
```

which makes rounded image. after upgrading to 4.7.0.968 (same with 4.7.0.1080) I found , that on IOS I am having squared image (on Android ok ).
reverting back to 4.6.0.800 , got rounded image

- Version with issue:
at least 4.7.0.968 and 4.7.0.1080
- Last known good version:
4.6.0.800
- IDE:
Visual studio for Mac 8.6.5.23
Xamarin.Mac 6.18.0.23 (d16-6 / 088c73638)

### Screenshots
![image](https://user-images.githubusercontent.com/45285052/86321561-e939d900-bc41-11ea-816d-e2616fa886ae.png)
![image](https://user-images.githubusercontent.com/45285052/86321746-47ff5280-bc42-11ea-9a10-744dcd23d04f.png)

between screenshots only Xamarin.forms nuget package was updated from 4.6.0.800 to 4.7.0.968

**Comment by jsuarezruiz** [comment]
Could be the same as https://github.com/xamarin/Xamarin.Forms/issues/11091
In this case, will be fixed by https://github.com/xamarin/Xamarin.Forms/pull/11129

**Comment by ish-1313** [comment]
it seems that for quick workaround the easiest workaround would be to use imagebutton instead frame+image

```
<!--
                            <Frame Grid.Row="0" Grid.Column="0"  IsClippedToBounds="True" CornerRadius="50"   Padding="0" 
                                                   HorizontalOptions="Fill" VerticalOptions="Fill">
                                <Image   Source="{Binding BindingImageSource}" Style="{StaticResource PetImageStyle}"  />
                            </Frame>
                            -->
                            <ImageButton Grid.Row="0" Grid.Column="0" CornerRadius="50" WidthRequest="100" HeightRequest="100"
                                         Source="{Binding BindingImageSource}" IsEnabled="False" Aspect="AspectFill" />
```
result

![image](https://user-images.githubusercontent.com/45285052/86351056-4f3b5600-bc6c-11ea-9529-4cdd99a3561c.png)

**Comment by samhouts** [comment]
closed by #11129
